### ISSUE TITLE
[NTR] Leydig stem cell

### ISSUE THREAD

**Issue by paulacalipho** [obophenotype/cell-ontology#1414]
**Preferred term label:** Leydig stem cell

**Synonyms**: stem Leydig cells

**Definition**: The adult Leydig cell population develops from undifferentiated mesenchymal-like stem cells present in the interstitial compartment of the neonatal testis. Distinct stages of development have been identified: Leydig stem cells, progenitor Leydig cells, immature Leydig cells, and adult Leydig cell. The Leydig stem cells are capable of self-renewal and also of differentiation to steroidogenic cells. [PMID: 21181888]

**Parent cell type term:** stem cell (CL:0000034)

**What is the anatomical structure that the cell is a part of?**:  Leydig cell region of testis (UBERON:0005212)

**Your nano-attribution (ORCID)**: 0000-0002-0819-0473

**Any additional notes or concerns**: I am not sure if add the Relationship: is_a: to CL:0000178 (Leydig cell) because CL:0000178 seems to refer to adult Leydig cell

**Comment by paolaroncaglia** [comment]
@paulacalipho 

> **Preferred term label:** Leydig stem cell
> (...)
> **Any additional notes or concerns**: I am not sure if add the Relationship: is_a: to CL:0000178 (Leydig cell) because CL:0000178 seems to refer to adult Leydig cell

Yes, CL:0000178 (Leydig cell) represents the differentiated, adult, testosterone-secreting cell type.
- [ ] The appropriate link to add would be 
CL:0000178 (Leydig cell) 'develops from' some CL:NEW (Leydig stem cell). I'll do that once the new term is created.

**Comment by paulacalipho** [comment]
@paolaroncaglia Many thanks
